CallsheetLocalizations/de.xcloc/Localized Contents/de.xliff

```diff
@@ -1611,7 +1611,7 @@ Sie enthält derzeit %2$lld pins.</target>
       </trans-unit>
       <trans-unit id="Show movies, shows, or both in credit lists." xml:space="preserve">
         <source>Show movies, shows, or both in credit lists.</source>
-        <target state="translated">Zeige Filme, TV Sendungen oder beides in der Liste der Credits an.</target>
+        <target state="translated">Zeige Filme, Serien oder beides in der Liste der Credits an.</target>
```

Comment:
"TV Sendungen" means TV broadcasts (like "The Price is Right"). "Serien" means TV shows (like "Shrinking").